### DISCUSSION TITLE
How do I disable Yetiforce shop notification?

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#16539] [How to] [answered]
Hello,

Currently I have not purchased the YetiForce Webservice Premium plugin, however, every time I start the section I get the notification "Paid features have been enabled in the YetiForce Webservice Premium system, please purchase a subscription."

![notificacion Yetiforce shop](https://user-images.githubusercontent.com/96985476/193369630-5035e3d7-efe1-44b2-9956-77f90cbc3f8f.jpg)

**Comment by mariuszkrzaczkowski** [comment]
if you don't use it, remove webservice application

**Reply by krseguros** [reply]
The point is that I don't have the web application [Webservice Premium], only the Webservice Standard

Webservice Standard
![image](https://user-images.githubusercontent.com/96985476/193411440-9d7827fe-0821-4607-a67c-842c1cbcdc9b.png)
Webservice Premium
![image](https://user-images.githubusercontent.com/96985476/193411464-e2c1e81d-2340-467d-82a6-925b11e1d3a0.png)
![image](https://user-images.githubusercontent.com/96985476/193411495-60823faa-c1ba-4abf-af46-92aa3826819a.png)

**Comment by mariuszkrzaczkowski** [comment]
For subscription issues, please contact sla@yetiforce.com

**Comment by rskrzypczak** [comment, marked as answer]
Go to the Web service - Applications panel and remove all entries of type Webservice Premium.
![image](https://user-images.githubusercontent.com/10221999/193985767-706b5134-422f-4a62-8b22-d4197ca06c81.png)

If you still see any messages after this, go to Company details panel and click Check status button.
![image](https://user-images.githubusercontent.com/10221999/193985783-86cbd0d4-62d4-4e5b-93d0-957a864a3a28.png)

**Reply by krseguros** [reply]
I did what was indicated but it still shows me the notification at every start, I do not have active Premium Webservice

![image](https://user-images.githubusercontent.com/96985476/194148783-6034e92a-4306-49fd-a534-574d6c74a64a.png)

![image](https://user-images.githubusercontent.com/96985476/194149158-fc3269e1-ab1a-4279-97f1-338cf4464edb.png)

**Reply by mariuszkrzaczkowski** [reply]
Is cron working properly?

**Reply by krseguros** [reply]
Yes

![image](https://user-images.githubusercontent.com/96985476/194150167-1b0c8f4e-c9d0-4f49-87cc-094572d7bd76.png)

**Reply by mariuszkrzaczkowski** [reply]
Show the result of this query:
```sql
SELECT `id`,`status`,`type`,`name` FROM w_yf_servers
```

**Reply by krseguros** [reply]
I don't understand exactly what you mean, and where do I see this?

**Reply by krseguros** [reply]
Greetings, Thank you for your help in slowing down this notification.
